cmd/root.go

```diff
@@ -37,6 +37,12 @@ var cfgFile string
 //Global variable with the port name
 var morserinoPortName string
 
+//Global variable with the debug level
+var morserinoDebugLevel string
+
+//Global variable with the name of the log file (if defined)
+var morserinoDebugFilename string
+
```

Comment:
- You can use the multi-line `var` syntax here, which allows to group symbols together, preferably semantically.
- Also, I don't see why we need to repeat `morserino` here. It doesn't bring much information, does it ?
- What does `debugLevel` means ? Shouldn't it be `logLevel` ? 
- Those variable names are pretty self descripting. The comments only repeat what is already stated by the variable. I suggest you remove them?

```suggestion
var (
       //Global variable with the debug level
        debugLevel string
       //Global variable with the name of the log file (if defined)
       debugFilename string
)
```